uint8 swarm_id
uint8 group_id
uint8 robot_id

